packages/amplify-e2e-tests/typings/aws-matchers.d.ts

```diff
@@ -2,5 +2,6 @@ namespace jest {
   interface Matchers<R> {
     toBeIAMRoleWithArn(roleName: string, arn?: string): R;
     toBeAS3Bucket(bucketName: string): R;
+    toHaveValidPolicyConditionMatchingIdpId(idpId: string): R;
```

Comment:
What's this change tied to?

Reply:
When this matcher was added it was not added to .d.ts and TSC requires it to be present, e2e test package had build failures, I fixed those to make sure code is correct.